disk/assets/monitors/disk_space_empty_forecast.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "[disk] Device {{device.name}} is going to be full on host {{host.name}}",
+	"type": "metric alert",
```

Comment:
```suggestion
	"type": "query alert",
```

Reply:
I'm not sure I understand why this would fail, but i don't think the type matters for the monitor itself, so i will accepth the suggestion